### DISCUSSION TITLE
Add a "Share" icon

### DISCUSSION THREAD

**Discussion by maxcountryman** [tailwindlabs/heroicons#618] [Ideas]
Now that we have the [Web Share API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Share_API), it'd be nice to have an icon for this express purpose a la the Apple suite.

<img width="50" alt="Screen Shot 2022-02-23 at 2 58 05 PM" src="https://user-images.githubusercontent.com/74351/155423635-48f5f70e-9753-4744-966f-33a80f3c888c.png">

**Comment by zefir-git** [comment]
A `share` icon exists already
![image](https://user-images.githubusercontent.com/66572326/156057118-6055c912-3f98-466e-a7d2-becb97d7d2f3.png)
```svg
<svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M8.684 13.342C8.886 12.938 9 12.482 9 12c0-.482-.114-.938-.316-1.342m0 2.684a3 3 0 110-2.684m0 2.684l6.632 3.316m-6.632-6l6.632-3.316m0 0a3 3 0 105.367-2.684 3 3 0 00-5.367 2.684zm0 9.316a3 3 0 105.368 2.684 3 3 0 00-5.368-2.684z" />
</svg>
```

**Reply by maxcountryman** [reply]
I don't think that has the same meaning (at least not as I've seen it used). I can't think of any iOS applications that use that icon to do Share API actions.

**Comment by alexander-schranz** [comment]
For me the share icon is also what william posted. The icon from apple looks like an upload icon which also exist just in a different form: 

<img width="398" alt="Bildschirmfoto 2022-04-02 um 23 33 57" src="https://user-images.githubusercontent.com/1698337/161401954-6d7f7f3c-ea0d-4dd0-bad5-3a852186bf47.png">
